VstsTasks/AzureDtlCreateCustomImage/new-azuredtl-customimage.json

```diff
@@ -8,34 +8,8 @@
         "labName": {
             "type": "string"
         },
```

Comment:
github makes it really hard to understand the diff ... because we dont see the entire files here. Could you describe what was the bug, so that we can understand the fix.

Reply:
Before we used the `osType` to distinguish in the RP and choose either the `linuxOsState` or `windowsOsState`. We have since moved to simply check if the OS state object is present and default to the Windows OS State check first. So, if both objects are passed, we take Windows even though the OS Type was set to Linux. Therefore, I have change the logic in the task to ensure we only pass the corresponding OS State based on the OS Type. We do the same thing from the Azure Portal logic. I match that logic.